packages/syft/src/syft/service/worker/worker_pool_service.py

```diff
@@ -178,6 +179,109 @@ def delete_worker(
             message=f"Worker with id: {worker_id} deleted successfully from pool: {worker_pool.name}"
         )
 
```

Comment:
MAYBE: we could also add a test to our 10-container-images.ipynb to test the functionality being integrated.

Reply:
Added tests for get_worker and get_worker_status but left out test for delete_worker as the notebooks started failing afterwards.